tests/api.c

```diff
@@ -7481,13 +7593,40 @@ static void test_wolfSSL_set_options(void)
     AssertTrue((SSL_set_options(ssl, SSL_OP_NO_COMPRESSION) &
                                SSL_OP_NO_COMPRESSION) == SSL_OP_NO_COMPRESSION);
 
+		AssertTrue(SSL_set_msg_callback(ssl, msg_cb) == SSL_SUCCESS);
+		SSL_set_msg_callback_arg(ssl, arg);
+
+		AssertTrue(SSL_CTX_set_alpn_protos(ctx, protos, len) == SSL_SUCCESS);
```

Comment:
We should also check to make sure bad arguments correctly error out (ex: NULL pointers for ctx and protos).